src/codegen/value.cpp

```diff
@@ -195,7 +195,17 @@ Value Value::LogicalOr(CodeGen &codegen, const Value &o) const {
 
   return Value{GetType(), codegen->CreateOr(GetValue(), o.GetValue())};
 }
-
+// Is Null and Is Not Null
+Value Value::IsNull(CodeGen &codegen, int nulltesttype ) const {
+  Value isNull;
+  codegen::Value null_val = Type::GetNullValue(codegen, GetType());
+  if (nulltesttype == 0) {
+    isNull = CompareEq(codegen, null_val);
+  } else {
+    isNull = CompareNe(codegen, null_val);
+  }
+  return isNull;
+}
```

Comment:
The `if` logic is duplicated and can be removed. You can then remove the `nulltesttype` parameter. Also, change `isNull` to `is_null` to be consistent with naming.